### PR TITLE
[Cloudservice] `az cloud-service`: Add new command group `os-family` to support managing os family

### PR DESCRIPTION
**Pull request by Jing-song**
---
aaz model: https://github.com/Azure/aaz/pull/404

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Cloudservice

File: src/cloudservice/azext_cloudservice/aaz/latest/cloud_service/__cmd_group.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command_group(
+    "cloud-service",
+)
+class __CMDGroup(AAZCommandGroup):
+    """Manage cloudservice
```
Comment (yanzhudd) [inline src/cloudservice/azext_cloudservice/aaz/latest/cloud_service/__cmd_group.py:18]:
```suggestion
    """Manage cloud service
```

Comment (yanzhudd) [reply src/cloudservice/azext_cloudservice/aaz/latest/cloud_service/__cmd_group.py:18]: Please remove help messages of this command group in _help.py as well.

File: src/cloudservice/azext_cloudservice/aaz/latest/cloud_service/role_instance/_reimage.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "cloud-service role-instance reimage",
```
Comment (yanzhudd) [inline src/cloudservice/azext_cloudservice/aaz/latest/cloud_service/role_instance/_reimage.py:15]: Have we added test for this new added command?

File: src/cloudservice/azext_cloudservice/aaz/latest/cloud_service/_reimage.py [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "cloud-service reimage",
```
Comment (yanzhudd) [inline src/cloudservice/azext_cloudservice/aaz/latest/cloud_service/_reimage.py:15]: May I ask if we have added test for this new added command?

Comment (Jing-song) [reply src/cloudservice/azext_cloudservice/aaz/latest/cloud_service/_reimage.py:15]: It is available in cmd, but there are no resources to test it.

File: src/cloudservice/azext_cloudservice/aaz/latest/cloud_service/role_instance/_reimage.py (hunk 2 above)
Comment (Jing-song) [reply src/cloudservice/azext_cloudservice/aaz/latest/cloud_service/role_instance/_reimage.py:15]: It is available in cmd, but there are no resources to test it.

**Comment by yanzhudd** [comment]
If the newly added help messages already exists in _help.py, please remove that duplicated contents of _help.py.

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**